Extension/src/LanguageServer/extension.ts

```diff
@@ -1161,6 +1169,8 @@ function handleCrashFileRead(err: NodeJS.ErrnoException | undefined | null, data
 }
 
 export function deactivate(): Thenable<void> {
+    // Log timestamps telemetry.
```

Comment:
>// Log timestamps telemetry. 
Is this comment for the line `clients.timeTelemetryCollector.clear();`? It doesn't match what the function `clear()` is doing.

Reply:
I changed the functionality but forgot to change the comment part.